src/ResourceManager/Sql/Commands.Sql/Auditing/Cmdlet/SqlDatabaseAuditingCmdletBase.cs

```diff
@@ -86,6 +86,14 @@ protected override AuditingPolicyModel PersistChanges(AuditingPolicyModel model)
             return null;
         }
 
+        /// <summary>
+        /// Execute the cmdlet
+        /// </summary>
+        protected override void ProcessRecord()
+        {
+            SqlDatabaseServerAuditingCmdletBase.PrintDeprecationMessageForAuditingCmdlets(this);
```

Comment:
Why is this needed? I thought you could put [Obsolete] attribute on the old cmdlets and that will automatically print deprecation warning.